### DISCUSSION TITLE
UI Developer

### DISCUSSION THREAD

**Discussion by hodanoori** [WeAreMahsaAmini/Core#479] [Introductions]
Hi, I'm UI developer. Let me know if I can help you there. I can spend an hour per day on this project. Thanks and best regards, Hoda

**Comment by tcf-admin** [comment]
Hi @hodanoori, 
please check out forDem's introduction here: https://github.com/WeAreMahsaAmini/Core/discussions/495
Hope you find it interesting and we can have you next to us.
